src/Form/ThirdPartyServicesForm.php

```diff
@@ -88,10 +85,11 @@ public function buildForm(array $form, FormStateInterface $form_state, array &$i
       $ga_config = $this->configFactory->get('google_analytics.settings');
       $form['google_analytics_account'] = [
         '#default_value' => $ga_config->get('account'),
-        '#description' => $this->t('This ID is unique to each site you want to track separately, and is in the form of UA-xxxxxxx-yy. To get a Web Property ID, <a href=":analytics">register your site with Google Analytics</a>, or if you already have registered your site, go to your Google Analytics Settings page to see the ID next to every site profile. <a href=":webpropertyid">Find more information in the documentation</a>.', [':analytics' => 'http://www.google.com/analytics/',
-          ':webpropertyid' => Url::fromUri('https://developers.google.com/analytics/resources/concepts/gaConceptsAccounts', ['fragment' => 'webProperty'])
-            ->toString()
-        ]),
+        '#description' => $this->t('This ID is unique to each site you want to track separately, and is in the form of UA-xxxxxxx-yy. To get a Web Property ID, <a href=":analytics">register your site with Google Analytics</a>, or if you already have registered your site, go to your Google Analytics Settings page to see the ID next to every site profile. <a href=":webpropertyid">Find more information in the documentation</a>.',
+          [
+            ':analytics' => 'http://www.google.com/analytics/',
```

Comment:
URL should be in config, please. Google used to change its API frequently, no need to put us in legacy code mode

Reply:
This is old code, I just updated code style and this is out of scope for the search. I can make issue on GitHub and fix it separately